website/content/en/preview/concepts/deprovisioning.md

```diff
@@ -142,28 +142,25 @@ data:
 
 ## Drift
 
-Karpenter Drift will classify each CRD field as a (1) Static, (2) Dynamic, or (3) Behavioral field and will treat them differently. Static Drift will be a one-way reconciliation, triggered only by CRD changes. Dynamic Drift will be a two-way reconciliation, triggered by machine/node/instance changes and Provisioner or AWSNodetemplate changes.
+Most felids are implemented one-way reconciliation, triggered only by CRD changes. Some special cases will be a two-way reconciled, triggered by machine/node/instance changes and Provisioner or AWSNodetemplate changes. For One-way Reconciliation, values in the CRDs are reflected in the machine in the same way that they’re set. A machine will be detected as drifted if the values in the CRDs do not match the values in the machine. By default, fields are drifted using One-way Reconciliation. 
```

Comment:
some nits on grammar, capitalization. Happy to capitalize one-way or two-way or reconciliation, but should be consistent throughout.
```suggestion
Most fields are only triggered by CRD changes. Some special cases will be reconciled two-ways, triggered by machine/node/instance changes and Provisioner or AWSNodeTemplate changes. For one-way reconciliation, values in the CRDs are reflected in the machine in the same way that they’re set. A machine will be detected as drifted if the values in the CRDs do not match the values in the machine. By default, fields are drifted using one-way reconciliation. 
```